src/outcome_spaces/ordinal_patterns.jl

```diff
@@ -12,18 +12,19 @@ Subtypes must implement fields:
 - `lt::Function`: A function determining how ties are to be broken when constructing
     permutation patterns from embedding vectors.
 """
-abstract type PermutationOutcomeSpace{m} <: CountBasedOutcomeSpace end
-const PermProbEst = PermutationOutcomeSpace
+abstract type OrdinalOutcomeSpace{m} <: CountBasedOutcomeSpace end
 
 ###########################################################################################
 # Types and docstrings
 ###########################################################################################
 """
     OrdinalPatterns <: OutcomeSpace
-    OrdinalPatterns(; m = 3, τ = 1, lt::Function = ComplexityMeasures.isless_rand)
+    OrdinalPatterns{m}(τ = 1, lt::Function = ComplexityMeasures.isless_rand)
```

Comment:
Is it intentional that `τ` and `lt` are now positional arguments? I think we should keep them as keyword arguments, unless there is a good reason to not do so.